common/src/app/common/path/bool.cljc

```diff
@@ -256,7 +256,7 @@
 
 (defn fix-move-to
   [content]
-  ;; Remove the field `:prev` and makes the necesaries `move-to`
+  ;; Remove the field `:prev` and makes the necessaries `move-to`
```

Comment:
This seems wrong, but I'm not sure what it's trying to say.

Reply:
I have asked the author, and your fix is correct, and the whole phrase is also correct. Ty